src/cozmo/annotate.py

```diff
@@ -238,15 +267,22 @@ def apply(self, image, scale):
             if scale != 1:
                 box *= scale
             add_img_box_to_image(image, box, self.box_color, text=text)
+            add_polygon_to_image(image, obj.left_eye, scale, self.box_color)
+            add_polygon_to_image(image, obj.right_eye, scale, self.box_color)
+            add_polygon_to_image(image, obj.nose, scale, self.box_color)
+            add_polygon_to_image(image, obj.mouth, scale, self.box_color)
 
     def label_for_face(self, obj):
         '''Fetch a label to display for the face.
 
         Override or replace to customize.
         '''
+        expression = obj.known_expression
+        if len(expression) > 0:
+            expression += " "
         if obj.name:
-            return ImageText(obj.name)
-        return ImageText('(unknown face %d)' % obj.face_id)
+            return ImageText('%s%s (%d)' % (expression, obj.name, obj.face_id))
+        return ImageText('(unknown%s face %d)' % (expression, obj.face_id))
```

Comment:
no, that's deliberate - if the string is empty (e.g. face expression estimation is off) then we don't want to add a stray space in